toxencryptsave/toxencryptsave.h

```diff
@@ -190,59 +222,66 @@ typedef struct {
  *
  * returns true on success
  */
-bool tox_derive_key_from_pass(const uint8_t *passphrase, size_t pplength, TOX_PASS_KEY *out_key,
-                              TOX_ERR_KEY_DERIVATION *error);
+bool tox_pass_key_derive_from_pass(struct Tox_Pass_Key *_key, const uint8_t *passphrase, size_t pplength,
```

Comment:
>`tox_pass_key_derive_from_pass()`

Redundant department of redundant redundancy is calling. This function doesn't make much sense as far as the name goes.